.clj-kondo/macros/metabase/task/setup/query_analysis_setup.clj

```diff
@@ -8,5 +8,6 @@
          ~(macros.common/ignore-unused 'c2) 2
          ~(macros.common/ignore-unused 'c3) 3
          ~(macros.common/ignore-unused 'c4) 4
-         ~(macros.common/ignore-unused 'arch) 5]
```

Comment:
the time is becoming right to draw this into a macro :face_with_spiral_eyes:

Reply:
Code reuse in kondos is a bit tricky, but I feel you.